maven-plugin/src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -712,6 +712,10 @@ public void setContainerId(String containerId) {
         setParam(PROPERTY_DOCKER_MYSQL, containerId);
     }
 
+    public void setPropertyValue(String propertyName, String value){
+        setParam("property."+propertyName+".default", value);
```

Comment:
Shouldn't you be calling setParam("property." + propertyName, value) to be concise with distro.properties?